draft-ietf-taps-interface.md

```diff
@@ -2992,25 +2992,25 @@ Connection.EndBatch()
 ### Send on Active Open: InitiateWithSend {#initiate-and-send}
 
 For application-layer protocols where the Connection initiator also sends the
-first message, the InitiateWithSend() action combines Connection initiation with
+first Message, the `InitiateWithSend` action combines Connection initiation with
 a first Message sent:
 
 ~~~
 Connection := Preconnection.InitiateWithSend(messageData, messageContext?, timeout?)
 ~~~
 
-Whenever possible, a messageContext should be provided to declare the Message passed to InitiateWithSend
+Whenever possible, a messageContext should be provided to declare the Message passed to `InitiateWithSend`
```

Comment:
```suggestion
Whenever possible, a MessageContext should be provided to declare the Message passed to `InitiateWithSend`
```
MessageContext is also an object… do we need to double-check this too?

Reply:
Fixed up throughout